src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,53 @@
+.container {
+  padding: 0 20px;
+
+  @include tablet {
+    padding: 0 27px;
+  }
+
+  @include desktop {
+    padding: 0 50px;
+    margin: auto;
+  }
+
+  @include big-desktop {
+    padding: 0 130px;
+    margin: auto;
+  }
+}
+
+html {
+  font-family: "Poppins", sans-serif;
+  font-weight: 400;
+  line-height: 150%;
+  font-size: 16px;
+  scroll-behavior: smooth;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.title {
```

Comment:
The same goes for '.title'. This is a specific class and should not be styled in the normalize file. Consider moving it to a relevant component file.